docs/quickstarts/excel-custom-functions-quickstart.md

```diff
@@ -80,6 +80,31 @@ In your Excel workbook, try out the `ADD` custom function by completing the foll
 
 The `ADD` custom function computes the sum of the two numbers that you specify as input parameters. Typing `=CONTOSO.ADD(10,200)` should produce the result **210** in the cell after you press enter.
 
+If the `CONTOSO` namespace isn't available in the autocomplete menu, take the following steps to register the add-in in Excel.
+
+### [Excel on Windows or Mac](#tab/excel-windows)
+
+1. In Excel, choose the **Insert** tab and then choose the down-arrow located to the right of **My Add-ins**.
+    ![Screenshot of the Insert ribbon in Excel on Windows, with the My Add-ins down-arrow highlighted.](../images/select-insert.png)
```

Comment:
I think the docs linter recommends a blank line between text blocks and images. As long as the image statement is indented, it'll still show up under the numbered list point.